client/tasks/task-list.tsx

```diff
@@ -26,6 +31,28 @@ export const getEventPrefix = ( id: string ): string => {
 	return `${ id }_tasklist_`;
 };
 
+export const taskSortByKey = ( keys: TaskSortType[] ) => (
+	a: TaskType,
+	b: TaskType
+) => {
+	if ( ! keys.length ) {
+		return 0;
+	}
+	if ( keys.includes( 'isComplete' ) && ( a.isComplete || b.isComplete ) ) {
+		return a.isComplete ? 1 : -1;
+	}
+	// Three is the lowest level.
+	if ( keys.includes( 'level' ) ) {
+		const aLevel = a.level || 3;
+		const bLevel = b.level || 3;
+		if ( aLevel === bLevel ) {
+			return 0;
+		}
+		return aLevel > bLevel ? 1 : -1;
+	}
+	return 0;
+};
```

Comment:
For now I used this approach, if tasks sorting becomes more comprehensive, we might want to create an object defining the 'asc' or 'desc' order for each type.

Reply:
I'm not sure if it adds a lot of work to this PR, but I'd be in favor of including that logic now if possible.